group.yml

```diff
@@ -516,13 +516,13 @@ repos:
     reposync:
       enabled: true
 
-  rhel-9.6-nfv:
+  rhel-96-nfv:
     conf:
       baseurl:
         x86_64: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/x86_64/os
-        aarch64: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/aarch64/os
-        ppc64le: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/ppc64le/os
-        s390x: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/s390x/os
+        aarch64: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/x86_64/os
+        ppc64le: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/x86_64/os
+        s390x: https://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.6/compose/NFV/x86_64/os
```

Comment:
Is this correct? All arches have a URI for `x86_64`. 
@lgarciaaco @joepvd @sdodson

Reply:
if not, @bryan-cox created a fix in #6007

Reply:
Yes, this is correct. The urls for the non-x86_64 do not exist. We need to list them here because our tooling expects a non-404-ing link there.

Reply:
@devguyio What problem are you and @bryan-cox trying to solve?

Reply:
Red herrings in CI build logs were pruned with https://github.com/openshift/release/pull/60743